scripts/map_to_lines.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python2
```

Comment:
Could use a tiny "howto", even if it's just a one line description and how to run the help command.